src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -134,17 +134,11 @@ class BankAccountStep extends React.Component {
             return;
         }
 
-        BankAccounts.setupWithdrawalAccount({
-            acceptTerms: this.state.hasAcceptedTerms,
-            accountNumber: this.state.accountNumber,
-            routingNumber: this.state.routingNumber,
-            setupType: CONST.BANK_ACCOUNT.SETUP_TYPE.MANUAL,
-
-            // Note: These are hardcoded as we're not supporting AU bank accounts for the free plan
-            country: CONST.COUNTRY.US,
-            currency: CONST.CURRENCY.USD,
-            fieldsType: CONST.BANK_ACCOUNT.FIELDS_TYPE.LOCAL,
-        });
+        BankAccounts.connectBankAccountManually(
+            this.state.accountNumber,
+            this.state.routingNumber,
+            '',
```

Comment:
In what cases is this not blank?

Reply:
I'm waiting on the PR for the automated flow (this is the manual one) to fully implement this. For Chase bank accounts, we need to ask people to enter the info manually afterwards (through this flow). In those cases, we'll pass the info we got from the automated flow in this parameter, so we can check the manually entered one is correct. It is basically a corner-ish case.